src/main.py

```diff
@@ -222,7 +224,7 @@ def componentLifeCashflow(comp, cf, variables, lifetimeCashflows, v=100, pyomoSw
           name = orig
         else:
           name = '(from input)'
-        if pyomoSwitch == None:
+        if pyomoSwitch == False:
```

Comment:
since it's boolean, it's more standard to say `if pyomoSwitch:` than check equality to False.

Reply:
this shows up a few times in these changes

Reply:
Do you mean:
if pyomoSwitch:
  if pyomoSwith == False:
    ...........

Reply:
Given:
```python
myVar = False
```
then
```python
if myVar:
  print('no equals signs')
if myVar == False:
 print('with equals signs')
```
both print out, and the former is preferred (no equals signs). `None` is slightly different, in that case the preferred check is
```python
myVar = None

if myVar is None:
  print('is none')
```
see https://www.python.org/dev/peps/pep-0008/